### ISSUE TITLE
The input help description of nvda +ctrl +space should be rewritten to present the same information as the summary in the User Guide entry. 

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#13344]
### Steps to reproduce:
1. Press NVDA+1 to open input help mode;
2. Then press NVDA+Control+Space;
3. Finally, press NVDA+1 to leave the input help.

### Actual behavior:
We heard the following in Step 2:  
"Moves the focus to the next closest document that contains the focus."
Sorry this description confused me, It seems that others also think this should be clarified.
[see here](https://nvda.groups.io/g/nvda/topic/under_what_circumstances_does/88946624?p=,,,50,0,0,0::recentpostdate/sticky,,,50,2,0,88946624,previd=1644972182590922761,nextid=1644078371290568461&previd=1644972182590922761&nextid=1644078371290568461)

### Expected behavior:
In the "6.4. Embedded Objects" section of the user guide, we can see the following summary:
"Moves the focus out of the current embedded object and into the document that contains it"

Additional Consideration:
This gesture seems like it should only work in browse mode, but for now it's a global gesture.

#### NVDA installed/portable/running from source:
installed and portable

#### NVDA version:
2021.3.1
#### Windows version:
all version

#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
cc: @Qchristensen
